test/index.js

```diff
@@ -1,245 +0,0 @@
-require('should');
-var Limiter = require('..');
```

Comment:
let's keep the old tests for now, I want to make sure that we have backward compatibility